src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/SchedularService.kt

```diff
@@ -15,4 +16,11 @@ class SchedularService(
       metricsService.recordPrisonersCountForEachPrison()
     }
   }
+
+  @Scheduled(initialDelay = 30000, fixedDelay = Long.MAX_VALUE)
+  fun oneTimeScheduledTask() {
```

Comment:
Im happy with the approach but we need to make sure we remove it once the data in the prisoner table is updated